### PR TITLE
feat: handle normalized asset service

### PR DESCRIPTION
**Pull request by 0xApotheosis**
## Description

Update `web` to handle the normalization to the `asset-service` made in https://github.com/shapeshift/lib/pull/712 (broken into https://github.com/shapeshift/lib/pull/743, https://github.com/shapeshift/lib/pull/744, https://github.com/shapeshift/lib/pull/745, https://github.com/shapeshift/lib/pull/746).

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

https://github.com/shapeshift/lib/issues/711

Closes https://github.com/shapeshift/lib/issues/651

## Risk

This touches a lot of places relating to assets and accounts.

A full regression test should be performed before merging.

## Testing

A regression test, with a particular focus on viewing assets and accounts, should be performed before merging.

## Screenshots (if applicable)

N/A

### CODE REVIEW

File: src/state/slices/marketDataSlice/marketDataSlice.tsx
Comment (0xApotheosis) [inline src/state/slices/marketDataSlice/marketDataSlice.tsx:80]: Handle breaking changes in https://github.com/shapeshift/lib/pull/741

File: src/components/AccountAssets/AccountAssets.tsx
Comment (0xdef1cafe) [inline src/components/AccountAssets/AccountAssets.tsx:25]: use asset reference rather than eth?

File: src/context/AppProvider/AppContext.tsx
Comment (0xdef1cafe) [inline src/context/AppProvider/AppContext.tsx:173]: we can remove the whole bitcoin shit check above now too

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]
nit picks

File: src/components/AccountAssets/AccountAssets.tsx
Comment (0xApotheosis) [reply src/components/AccountAssets/AccountAssets.tsx]: Addressed in https://github.com/shapeshift/web/pull/1908/commits/7cd8a1b29941e6cf011fe4054a4052058314b857

File: src/context/AppProvider/AppContext.tsx
Comment (0xApotheosis) [reply src/context/AppProvider/AppContext.tsx:158]: Addressed in https://github.com/shapeshift/web/pull/1908/commits/7cd8a1b29941e6cf011fe4054a4052058314b857

**Comment by MBMaria** [comment]
Did regression testing on this PR. ETH sends are coming up with an insufficient funds error no matter the size of the send. 
All other TX working fine. ERC-20 send and receive, ETH trade, ERC-20 trade, stake and unstake had no issues. 

To recreate the ETH send issue, Select ETH in wallet and go to send button at top right of screen. Enter in valid ETH address or scan QR code, select next. Attempt to enter any amount into the input box (doesn't matter if the value is in ETH or USD or even a . ) and notice the "Insufficient Funds" error. 
image

**Comment by 0xApotheosis** [comment]
Putting into draft as there is a couple of items from ops for me to look at tomorrow (likely regressions).

**Comment by gomesalexandre** [comment]
This has operations approval with the current state.
Still needs https://github.com/shapeshift/lib/pull/777 to be published, and the `@shapeshiftoss/asset-service` dependency to be bumped here before this can get merged.
Opening this for early review.

**Comment by gomesalexandre** [comment]
Will reopen when https://github.com/shapeshift/lib/pull/789 is published

**Comment by 0xdef1cafe** [comment]
tested by ops here https://discord.com/channels/554694662431178782/984831137333526538

**Review by gomesalexandre — APPROVED** [review]

**Comment by 0xApotheosis** [comment]
Giving this one final sanity check before merging.

**Pull request merged**
